test/testenv/secretutil.go

```diff
@@ -69,33 +68,32 @@ func GetSecretObject(deployment *Deployment, ns string) *SecretResponse {
 }
 
 // GetSecretKey Gets the value to specific key from secret object
-func GetSecretKey(deployment *Deployment, ns string, key string) string {
-	restResponse := GetSecretObject(deployment, ns)
+func GetSecretKey(deployment *Deployment, ns string, key string, secretName string) string {
+	restResponse := GetSecretObject(deployment, ns, secretName)
 	//return key based on request
 	switch key {
 	case "hec_token":
-		key := DecodeBase64(restResponse.HecToken)
-		return key
+		value := DecodeBase64(restResponse.HecToken)
+		return value
 	case "idxc_secret":
-		key := DecodeBase64(restResponse.IdxcSecret)
-		return key
+		value := DecodeBase64(restResponse.IdxcSecret)
+		return value
 	case "pass4SymmKey":
-		key := DecodeBase64(restResponse.Pass4SymmKey)
-		return key
+		value := DecodeBase64(restResponse.Pass4SymmKey)
+		return value
 	case "password":
-		key := DecodeBase64(restResponse.Password)
-		return key
+		value := DecodeBase64(restResponse.Password)
+		return value
 	case "shc_secret":
-		key := DecodeBase64(restResponse.ShcSecret)
-		return key
+		value := DecodeBase64(restResponse.ShcSecret)
+		return value
 	default:
 		return "Invalid Key"
 	}
 }
 
 //ModifySecretObject Modifies the entire secret object
-func ModifySecretObject(deployment *Deployment, data map[string][]byte, ns string) bool {
-	secretName := fmt.Sprintf(SecretObjectName, ns)
+func ModifySecretObject(deployment *Deployment, data map[string][]byte, ns string, secretName string) bool {
```

Comment:
Add a log in to print which secret is being update.

Reply:
added